sycl/include/CL/sycl/handler.hpp

```diff
@@ -411,7 +411,7 @@ class __SYCL_EXPORT handler {
     using KI = sycl::detail::KernelInfo<KernelName>;
     // Empty name indicates that the compilation happens without integration
     // header, so don't perform things that require it.
-    if (KI::getName() != "") {
```

Comment:
Compiler had put the same pointer here as in https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/detail/kernel_desc.hpp#L59 or https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/detail/kernel_desc.hpp#L68, didn't it?

Reply:
I guess so. But it is UB.